media-picker/src/main/java/com/miguelgaeta/media_picker/MediaPicker.java

```diff
@@ -177,19 +178,19 @@ public static void startForImageCrop(final Provider provider, final File file, i
      */
     private static void startForImageCrop(final Provider provider, final Uri uri, int outputWidth, int outputHeight, int colorInt, final OnError onError) {
```

Comment:
need to get rid of `colorInt` and provide users ability to provide their own `UCrop.Options`.